lib/server/services/log/LogService.js

```diff
@@ -37,6 +37,7 @@ class LogService {
                 .include('tags')
                 .include('runs')
                 .include('subsystems')
+                .include('environments')
```

Comment:
You should not include environments by default, this is againts the purpose of fetching them lazilly.